package.json

```
@@ -74,6 +74,7 @@
     "redux": "^4.0.0",
     "redux-devtools-extension": "^2.13.7",
     "redux-saga": "^0.16.0",
+    "twin.macro": "1.2.0",
```

```suggestion
    "twin.macro": "^1.12.1",
```

get the latest 1-ish version?

---

ok